src/module/system/text-editor.ts

```diff
@@ -314,11 +314,32 @@ class TextEditorPF2e extends TextEditor {
                 });
             }
 
+            if (item) {
+                const itemTraits = (item as any).traits;
```

Comment:
For some reason traits does not exist on the interface but it does exist in practice.